lib/internal/Magento/Framework/GraphQl/Query/QueryProcessor.php

```diff
@@ -49,10 +55,11 @@ public function process(
         array $variableValues = null,
         string $operationName = null
     ) : array {
+        var_dump($this->exceptionFormatter->shouldShowDetail());
```

Comment:
Function or method 'var_dump()' is obsolete in file '/opt/bamboo/agents/agent1/xml-data/build-dir/MCCE23-L41453-SL/magento2/lib/internal/Magento/Framework/GraphQl/Query/QueryProcessor.php'.